_content/features/features.json

```diff
@@ -0,0 +1,3 @@
+{
+  "layout": "main.njk"
+}
```

Comment:
Since all of our features/*.md files are in a directory, we can specify a shared `layout` for them instead of specifying the `layout` in each template's front-matter; like above w/ `_content/about.md` or below on `_content/index.md`.